### ISSUE TITLE
Crash report on 7.1: CursorIndexOutOfBoundsException in MediaGridAdapter

### ISSUE THREAD

**Issue by hypest** [wordpress-mobile/WordPress-Android#5674]
```
Fatal Exception: android.database.CursorIndexOutOfBoundsException: Index -1 requested, with a size of 75
       at android.database.AbstractCursor.checkPosition(AbstractCursor.java:460)
       at android.database.AbstractWindowedCursor.checkPosition(AbstractWindowedCursor.java:136)
       at android.database.AbstractWindowedCursor.getString(AbstractWindowedCursor.java:50)
       at android.database.CursorWrapper.getString(CursorWrapper.java:137)
       at org.wordpress.android.ui.media.MediaGridAdapter$GridViewHolder$3.onClick(MediaGridAdapter.java:294)
       at android.view.View.performClick(View.java:5637)
       at android.view.View$PerformClick.run(View.java:22429)
       at android.os.Handler.handleCallback(Handler.java:751)
       at android.os.Handler.dispatchMessage(Handler.java:95)
       at android.os.Looper.loop(Looper.java:154)
       at android.app.ActivityThread.main(ActivityThread.java:6169)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:896)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:780)
```

**Comment by daniloercoli** [comment]
This should be fixed in 7.2 since we dropped the cursor in that release.
@nbradbury Can you confirm this?  What do you think on adding a `isValidPosition` here: https://github.com/wordpress-mobile/WordPress-Android/blob/10ef2405663bbe88e27a76c946ea7582252f7460/WordPress/src/main/java/org/wordpress/android/ui/media/MediaGridAdapter.java#L285

**Comment by nbradbury** [comment]
> This should be fixed in 7.2 since we dropped the cursor in that release.
> @nbradbury Can you confirm this?

Yes, that's correct - the cursor was dropped so this won't be in an issue any longer.

> What do you think on adding a isValidPosition here: 

Addressed in #5720.
